docs/docs/self-host/env.mdx

```diff
@@ -27,6 +27,12 @@ growthbook:
     - API_HOST=http://<my ip>:3100
 ```
 
+:::info Unique ports
```

Comment:
It's not quite unique ports. They can both run on port 443 for example, as long as they are on different domains.

Reply:
Updated copy in this commit 6bab3948bdb577d1058e528705e7a4cadef8e1aa